verification/curator-service/ui/cypress/integration/components/Curator.spec.ts

```diff
@@ -0,0 +1,306 @@
+            '2020/01/01',
+        );
+        cy.get('input[name="travelHistory[1].dateRange.end"]').should(
+            'have.value',
+            '2020/01/05',
+        );
+        cy.contains('Car');
+        cy.contains('Plane');
+        cy.contains('Business');
+        cy.contains('Yes');
+        // Pathogens.
+        cy.contains('Bartonella');
+        cy.contains('Ebola');
+        cy.get('textarea[name="notes"]').should(
+            'have.value',
+            'test notes\non new line',
+        );
+        // Transmission.
+        cy.contains('Airborne infection');
+        cy.contains('Assisted Living');
+        cy.contains('testcaseid12345678987654');
+        cy.contains('testcaseid12345678987655');
+        // Change a few things.
+        cy.get('div[data-testid="sex"]').click();
+        cy.get('li[data-value="Male"').click();
+        // Submit the changes.
+        cy.get('button[data-testid="submit"]').click();
+        cy.contains('Case edited');
+        // Updated info should be there.
+        cy.visit('/cases');
+        cy.contains('No records to display').should('not.exist');
+        cy.contains('Male');
+        // What's untouched should stay as is.
+        cy.contains('Asian');
+        // View full details about the case
+        cy.get('button[title="View this case details"]').click();
+        // Case data.
+        cy.contains('www.example.com');
+        cy.contains('superuser@test.com');
+        cy.contains('test notes on new line');
+        // Demographics.
+        cy.contains('21');
+        cy.contains('Male');
+        cy.contains('Accountant');
+        cy.contains('Afghan, Albanian');
+        cy.contains('Asian');
+        cy.contains('France');
+        cy.contains('45.7589');
+        cy.contains('4.8414');
+        // Events.
+        cy.contains('2019-12-31');
+        cy.contains('2020-01-01');
+        cy.contains('2020-01-02');
+        cy.contains('2020-01-03');
+        cy.contains('2020-01-04');
+        cy.contains('2020-01-05');
+        cy.contains('2020-01-06');
+        cy.contains('PCR test');
+        cy.contains('Recovered');
+        cy.contains('Yes');
+        //  Symptoms.
+        cy.contains('dry cough, mild fever');
+        // Preexisting conditions.
+        cy.contains('ABCD syndrome, ADULT syndrome');
+        // Transmission.
+        cy.contains('Airborne infection');
+        cy.contains('Assisted Living');
+        cy.contains('testcaseid12345678987654');
+        cy.contains('testcaseid12345678987655');
+        // Travel history.
+        cy.contains('Car, Plane');
+        cy.contains('Business');
+        cy.contains('Germany');
+        cy.contains('Bus');
+        cy.contains('United Kingdom');
+        // Pathogens and genome.
+        cy.contains('Bartonella (232), Ebola (41)');
+        cy.contains('www.example2.com');
+        cy.contains('test sequence name');
+        cy.contains('33000');
+        cy.contains('testSequenceId');
+    });
```

Comment:
You could add deleting the case for full CRUD flow

Reply:
in that case I am not convinced this gives us anything more than the existing test that deletes a case as it doesn't matter whether the case was created with the form or with fake data.